jobbergateapi2/tests/apps/job_scripts/test_routers.py

```diff
@@ -442,3 +448,115 @@ async def test_list_job_script_all(client, user_data, application_data, job_scri
     assert data[0]["id"] == 1
     assert data[1]["id"] == 2
     assert data[2]["id"] == 3
+
+
+@pytest.mark.freeze_time
+@pytest.mark.asyncio
+@database.transaction(force_rollback=True)
+async def test_update_job_script(client, user_data, application_data, job_script_data):
+    """
+    Test update job_script.
+    """
+    user = [UserCreate(id=1, **user_data)]
+    await insert_objects(user, users_table)
+
+    application = [Application(id=1, application_owner_id=1, **application_data)]
+    await insert_objects(application, applications_table)
+
+    job_scripts = [JobScript(id=1, **job_script_data)]
+    await insert_objects(job_scripts, job_scripts_table)
+
+    response = client.put("/job-scripts/1", data={"job_script_name": "new name"})
+
+    assert response.status_code == status.HTTP_201_CREATED
+    data = response.json()
+    now = datetime.now()
+
+    assert data["job_script_name"] == "new name"
+    assert data["id"] == 1
+    assert data["updated_at"] == now.isoformat()
+
+    query = job_scripts_table.select(job_scripts_table.c.id == 1)
+    job_script = JobScript.parse_obj(await database.fetch_one(query))
+
+    assert job_script is not None
+    assert job_script.job_script_name == "new name"
+    assert job_script.updated_at == now
+
+
+@pytest.mark.asyncio
+@database.transaction(force_rollback=True)
+async def test_update_job_script_not_found(client, user_data, application_data, job_script_data):
+    """
+    Test update a job_script not found, must return 404 and do nothing with the stored data.
```

Comment:
update this to use the long docstring format that we have used in other tests ... "This test proves that ...."